docs/source/environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - bioconda
   - conda-forge
 dependencies:
-  - hail>=0.2.58
+  - python=3.7
```

Comment:
`linear_regression` is not compatible with python3.6, which the environment installed by default